README.md

```diff
@@ -156,7 +157,8 @@ If you like this project please give it a start and share it with your friends.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-
+I would like to appreciate[Microverse](https://www.microverse.org/) for providing the resources and the platform for us to be a Professional full-stack developer.
+I would like to thank [sakwadesignstudio](https://www.behance.net/sakwadesignstudio) for providing the greate design in Behance that we used as a reference to build this Application 
```

Comment:
- [ ] [OPTIONAL] Very nice Acknowledgment you have here @MICHAELKITH  💪🏽 .
However, you could make it better by giving the Acknowledgment to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/)